app/model/des/FinancialData.scala

```diff
@@ -25,31 +25,23 @@ import play.api.libs.json.{Json, OFormat, Reads}
  * containing financial data for a VRN.
  */
 
-final case class FinancialData(idType: Option[String], idNumber: Option[String], regimeType: Option[String], processingDate: Option[String], financialTransactions: Option[Seq[Transaction]])
+final case class FinancialData(idType: String, idNumber: String, regimeType: String, processingDate: String, financialTransactions: Option[Seq[Transaction]])
```

Comment:
pleas add break lines

Reply:
I'm not sure but I think it would be possible to have:

`financialTransactions: Option[Seq[Transaction]]` programmed as:
`financialTransactions: Seq[Transaction]`

That woud solve dillema of difference between: `Some(Nil)` and `None`

Reply:
I've done the changed. Hopefully this is fine - probably would expect a 404 if there are no transactions.